ecosystem/plugin-search/src/client/composables/useHotKeys.ts

```diff
@@ -2,6 +2,7 @@ import { onBeforeUnmount, onMounted } from 'vue'
 import type { Ref } from 'vue'
 import type { HotKeyOptions } from '../../shared/index.js'
 import { isKeyMatched } from '../utils/index.js'
+import { isFocusingTextControl } from '../utils/isFocusingTextControl.js'
```

Comment:
We prefer to have this import from index.js, a refactor is needed.

Reply:
I’ve changed this now.